app/server/src/models/annotation/annotation.model.js

```diff
@@ -0,0 +1,89 @@
+const mongoose = require("mongoose");
+const { Body } = require("./body.model");
+const { Selector } = require("./selector.model");
+const { Target } = require("./target.model");
+
+const annotationSchema = new mongoose.Schema({
+  "@context": {
+    type: String,
+    required: true,
+  },
+  type: {
+    type: String,
+    required: true,
+  },
+  body: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Body",
+    },
+  ],
+  target: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Target",
+    required: true,
+  },
+  id: {
+    type: String,
+    required: true,
+  },
+});
+
+const Annotation = mongoose.model("Annotation", annotationSchema);
+
+const createAnnotation = async (context, type, body, target, id) => {
+  var bodies_l = [];
+  for (var body_el of body) {
+    var body_t = new Body({
+      purpose: body_el.purpose,
+      value: body_el.value,
+      type: body_el.type,
+      creator: body_el.creator,
+      created: body_el.created,
+    });
+    await body_t.save();
```

Comment:
can you please use createBody function of Body.

Reply:
Be aware that it is missing creator field and created field

Reply:
Done!